tests/signal/butter_test.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+import numpy as np
+import scipp as sc
+from scipp.signal import butter
+
+import pytest
+
+
+def array1d_linspace(*, coord=None):
+    if coord is None:
+        dim = 'xx'
+        size = 100
+        coord = sc.linspace(dim=dim, start=-0.1, stop=4.0, num=size, unit='m')
+    else:
+        dim = coord.dim
+        size = len(coord)
+    y = coord.copy()
+    y.unit = 'rad'
+    y = sc.sin(y)
+    rng = np.random.default_rng()
+    y.values += 0.1 * rng.normal(size=size)
+    return sc.DataArray(y, coords={dim: coord})
+
+
```

Comment:
`Wn` can be a 2 element array. Please add a test to ensure that our `butter` does not assume a scalar.

Reply:
See the `bandpass` test.

Reply:
Right. Never mind.